readme.md

```diff
@@ -1,6 +1,6 @@
 # Stopwatch task
 Replace `<your_account>` with your Github username and copy the links to Pull Request description:
-- [DEMO LINK](https://<your_account>.github.io/layout_stop-watch/)
+- [DEMO LINK](https://cyberte4.github.io/layout_stop-watch/)
```

Comment:
Add this link to the PR description
<img width="921" alt="image" src="https://github.com/mate-academy/layout_stop-watch/assets/64750958/45e831fa-a225-476c-829c-71d23e4994e6">